src/component/feature/FeatureToggleList/FeatureToggleList.jsx

```diff
@@ -23,43 +23,31 @@ import { useStyles } from './styles';
 import ListPlaceholder from '../../common/ListPlaceholder/ListPlaceholder';
 import { getCreateTogglePath } from '../../../utils/route-path-helpers';
 import { NAVIGATE_TO_CREATE_FEATURE } from '../../../testIds';
+import { resolveFilteredProjectId } from "../../../hooks/useFeaturesFilter";
```

Comment:
Hm. Why is prettier not taking effect here?

Reply:
Probably my IDE using the wrong style for auto-imports. Will fix. :+1: